docs/sampleApp.md

```diff
@@ -0,0 +1,56 @@
+## Deploying travelocity.com sample app
+
+To ensure you get the full understanding of configuring authenticator with WSO2 IS, the sample travelocity application is used in this use case. The samples run on the Apache Tomcat server and are written based on Servlet 3.0. Therefore, download Tomcat 7.x from [here](https://tomcat.apache.org/download-70.cgi).
+Install Apache Maven to build the samples. For more information, see [Installation Prerequisites](https://docs.wso2.com/display/IS570/Installation+Prerequisites).
+
+Follow the steps below to deploy the travelocity.com sample application:
+
+### Download the samples
+
+To be able to deploy a sample of Identity Server, you need to download it onto your machine first. 
+
+Follow the instructions below to download a sample from GitHub.
+
+* Create a folder in your local machine and navigate to it using your command line.
+
+* Run the following commands.
+  ```
+  mkdir is-samples
+  cd is-samples/
+  git init
+  git remote add -f origin https://github.com/wso2/product-is.git
+  git config core.sparseCheckout true
+  ```
+  
+* Navigate into the .git/info/ directory and list out the folders/files you want to check out using the echo command below.
+    ``` 
+    cd .git
+    cd info
+    echo "modules/samples/" >> sparse-checkout
+    ```
+    
+* Navigate out of .git/info directory and checkout the v5.4.0 tag to update the empty repository with the remote one.
+    ```
+    cd ..
+    cd ..
+    git checkout -b v5.4.0 v5.4.0
+    ```
+* Go to is-samples/modules/samples/sso/sso-agent-sample directory and run `mvn clean install` and get the war file from the target folder.
```

Comment:
It is better if we can say the name of the war file.

Reply:
improved